src/tz.cpp

```diff
@@ -697,12 +699,14 @@ load_timezone_mappings_from_xml_file(const std::string& input_path)
         }
     }
 
-    is.close();
+    //is.close(); Our stream cannot be closed, because it is not a file stream. 
     return mappings;
 }
 
-#endif  // _WIN32
+#endif  // _WIN32 
+#endif //!USE_OS_TZDB
```

Comment:
Is it worth me updating these comments to include the defined(RTC) condition?

Reply:
Probably not, reducing the potential for update conflicts is a bigger thing to go for.